modules/@apostrophecms/page/index.js

```diff
@@ -846,6 +847,20 @@ module.exports = {
   },
   handlers(self) {
     return {
+      '@apostrophecms/page-type:beforeSave': {
+        handleParkedSlugOverride(req, doc) {
+          if (!doc.parkedId) {
+            return;
+          }
+          const parked = self.parked.find(p => p.parkedId === doc.parkedId);
```

Comment:
This should apply to all parked properties outside of `_defaults`, not just `slug`.

Reply:
(slug is the immediate problem but won't be the last)